code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -95,7 +56,7 @@
 
 	var/total_part_size = return_total_size()
 	var/total_complexity = return_total_complexity()
-	var/HTML = list()
+	var/HTML = ""
```

Comment:
Congratulations on making this code worse.

Reply:
As if there is a way to make HTML interface code ported from Bay/Polaris noticeably worse.

Reply:
Amusing, but string concatenation does perform far worse than lists+Join(), a note for the future.